src/utility.ts

```diff
@@ -55,21 +55,18 @@ class Utility {
       }
       case "linux": {
         if (this.isXdgOpenAvailable()) {
-          spawnSync("xdg-open");
+          spawnSync(`xdg-open ${file}`);
```

Comment:
this probably works but I think it is safer to have the file as an argument with just `xdg-open` as the command

```suggestion
          spawnSync('xdg-open', file);
```
https://nodejs.org/api/child_process.html#child_processspawnsynccommand-args-options

Reply:
I fixed this, the args need to be an array.  It might have worked the other way, until it doesn't.  I also removed the special case opening a website in linux, turns out, it works the same on all platforms.